src/commissaire/constants.py

```diff
@@ -59,4 +59,6 @@
     'METHOD_NOT_FOUND': -32601,
     'INVALID_PARAMETERS': -32602,
     'INTERNAL_ERROR': -32603,
+    # Custom codes
+    'CONFLICT': -32049,
```

Comment:
nit: Apologies for totally splitting hairs, but this and future Commissaire-specific codes strike me more as "application defined errors", which according to the spec gives us the entire *positive* numeric space.  If a `CONFLICT` error is meant to correspond to an HTTP 409 status, we could just use HTTP status codes directly here.